node.gyp

```diff
@@ -747,6 +749,48 @@
   ], # end targets
 
   'conditions': [
+    [ 'node_target_type=="static_library"', {
+      'targets': [
+        {
+          'target_name': 'static_node',
+          'type': 'executable',
+          'product_name': '<(node_core_target_name)',
+          'dependencies': [
+            '<(node_core_target_name)',
+          ],
+          'sources+': [
+            'src/node_main.cc',
+          ],
+          'include_dirs': [
+            'deps/v8/include',
+          ],
+          'xcode_settings': {
+            'OTHER_LDFLAGS': [
+              '-Wl,-force_load,<(PRODUCT_DIR)/<(STATIC_LIB_PREFIX)'
+              '<(node_core_target_name)<(STATIC_LIB_SUFFIX)',
+            ],
+          },
+          'msvs_settings': {
+            'VCLinkerTool': {
+              'AdditionalOptions': [
+                '/WHOLEARCHIVE:<(PRODUCT_DIR)/lib/'
+                '<(node_core_target_name)<(STATIC_LIB_SUFFIX)',
+              ],
+            },
+          },
+          'conditions': [
+            ['OS in "linux freebsd openbsd solaris android aix"', {
+              'ldflags': [
+                '-Wl,--whole-archive,'
+                '<(OBJ_DIR)/<(STATIC_LIB_PREFIX)'
+                '<(node_core_target_name)<(STATIC_LIB_SUFFIX) ',
```

Comment:
Is the extra space intentional?

Reply:
It was a while since I did that but if I recall correctly this was required. I'll double check through. Thanks

Reply:
This was not required and I've now removed it.